src/super_gradients/training/models/detection_models/yolo_base.py

```diff
@@ -413,51 +351,32 @@ def _initialize_module(self):
             nms_iou = self.arch_params.nms_iou
             self._nms = YoloPostPredictionCallback(nms_conf, nms_iou)
 
-    def update_param_groups(self, param_groups: list, lr: float, epoch: int, iter: int,
-                            training_params: HpmStruct, total_batch: int) -> list:
-
-        lr_warmup_epochs = get_param(training_params, 'lr_warmup_epochs', 0)
-        if epoch >= lr_warmup_epochs:
-            return super().update_param_groups(param_groups, lr, epoch, iter, training_params, total_batch)
-        else:
-            return param_groups
-
     def _check_strides_and_anchors(self):
```

Comment:
Sorry for the review in waves, just noticed. Do you mind renaming this to `_check_strides` please?

Reply:
done https://github.com/Deci-AI/super-gradients/pull/300/commits/fcdab0aac5e6a19fb95a3de35b48b88f2a96edba